icubam/db/store.py

```diff
@@ -322,9 +336,9 @@ def disable_icu(self, manager_user_id: int, icu_id: int):
     self.enable_icu(manager_user_id, icu_id, False)
 
   def assign_user_as_icu_manager(self, admin_user_id: int, manager_user_id: int,
-                                 icu_id: int):
+                                 icu_id: int, force: bool = False):
```

Comment:
perhaps a little documentation on when it's fine to have force=true.

Reply:
you are right. done